routes/web.php

```diff
@@ -14,15 +15,22 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
+
+Route::controller(ShowController::class)->group(function() {
+    Route::get('/index', 'index')->name('index');
+    Route::get('/index/article/{id}', 'show')->name('article.show');
+});
+
+
 Auth::routes();
 Route::group(['middleware'=> 'auth'],function(){
     Route::controller(ArticleController::class)->group(function() {
-        Route::get('/index', 'index')->name('index');
-        Route::get('article/create', 'create')->name('article.create');
-        Route::post('article/create', 'store')->name('article.store');
-        Route::patch('article/{id}/update', 'update')->name('article.update');
-        Route::get('article/{id}/edit', 'edit')->name('article.edit');
-        Route::get('article/{id}', 'show')->name('article.show');
-        Route::delete('article/{id}/destroy', 'destroy')->name('article.destroy');
+        Route::get('/dashboard', 'dashboard')->name('dashboard');
+        Route::get('/dashboard/article/create', 'create')->name('article.create');
+        Route::post('/dashboard/article/create', 'store')->name('article.store');
+        Route::patch('/dashboard/article/{id}/update', 'update')->name('article.update');
+        Route::get('/dashboard/article/{id}/edit', 'edit')->name('article.edit');
+        Route::get('/dashboard/article/{id}', 'show')->name('dashboard.show');
```

Comment:
```suggestion
        Route::get('/dashboard/article/{id}', 'show')->name('dashboard.article.show');
```

全部ダブるからdashboard.article.*の形にした方がいいかも

Reply:
修正済。